docs/workforce-management-framework/topics/con-Security-Strategy-for-Keycloak.adoc

```diff
@@ -0,0 +1,28 @@
+[id='Security-Strategy-Keycloak-{chapter}']
+= Security Strategy for {WFM-RC-NameShort} {WFM-RC-KeycloakName}
+
+== Session Management
+{WFM-RC-KeycloakName} uses the Express middleware `Express Sessions` module for the {WFM-RC-NameShort} cloud server.
+Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
+session options.
+One of the key functions of {WFM-RC-KeycloakName} is that you can administer sessions on the
+link:{WFM-RC-KeycloakURL}server_admin/topics/sessions/administering.html[Keycloak server]
+
+== Access Control
+Keycloak has role based access control. This can be configured on the {WFM-RC-KeycloakName} server see
+link:{WFM-RC-KeycloakURL}server_admin/topics/roles.html[roles administration] for more information.
+
+== Password Storage
+On {WFM-RC-KeycloakName} passwords are not stored as clear text. Instead they are hashed using standard hashing algorithms before they are stored or validated.
```

Comment:
"On {WFM-RC-KeycloakName} passwords are not stored as clear text. Instead they are hashed using standard hashing algorithms before they are stored or validated. {WFM-RC-KeycloakName} allows to control passwords implementation by link"

**>**

"Keycloak uses the PBKDF2 algorithm for password hashing by default. {WFM-RC-KeycloakName} also provides enforced {WFM-RC-KeycloakURL}server_admin/topics/authentication/password-policies.html[password policies] from the server."

Small simplification.

Reply:
agree